gio/src/socket_control_message.rs

```diff
@@ -0,0 +1,38 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::prelude::*;
+use crate::SocketControlMessage;
+use glib::object::IsA;
+use glib::translate::*;
+
+impl SocketControlMessage {
+    #[doc(alias = "g_socket_control_message_deserialize")]
+    pub fn deserialize(level: i32, type_: i32, data: &[u8]) -> SocketControlMessage {
+        let size = data.len() as _;
+        unsafe {
+            from_glib_full(ffi::g_socket_control_message_deserialize(
```

Comment:
This can't fail?

Reply:
I'm quite sure it can

Reply:
It can, I filed a glib MR: https://gitlab.gnome.org/GNOME/glib/-/merge_requests/3099

Reply:
You can work around that in the meantime via `nullable = true` in the `Gir.toml`. Please add a link to the MR in there so we can get rid of it in the near future :)

Reply:
Ah you did.